### ISSUE TITLE
Vote on tweaks to be updated

### ISSUE THREAD

**Issue by fuzepod** [jlippold/tweakCompatible#21612]
A feature to vote for a tweak so that developers can see what tweaks the jailbreaking community would like updated first would help prioritize tweak development.

Related features include a top requested tweak to be updated and top tweaks that are already updated. Since you are already compiling a list of every tweak it would be great to also have a leaderboard of sorts showing the most popular voted tweaks. Just have a vote button for every tweak and then product two leaderboards: top voted tweaks that support x iOS version and top voted tweaks to be updated for x iOS version where x is the Cydia user's iOS version.

**Comment by jlippold** [comment]
My only issue with doing this is that this turns into a tweak review product instead of a compatibility reporter. People can be real dicks about reviews, and I'm not sure I want to have to censor reviews or deal with developers being upset about what someone said. 

I'll leave this open so more people can add to the discussion

**Comment by fuzepod** [comment]
Yes that would require moderating which would be a huge hassle and the project now is very conveniently automated. I wasn't thinking of text reviews though, more just a vote button and a counter on every tweak. The tweaks in Cydia already link the developers email if people want to give specific feedback.

**Comment by DatNightcat** [comment]
Yes please! There are so many tweaks I’ve seen in the past 8 years, and I’d love to have them back for iOS 11. None of my favorites work on iOS 11 (and I can’t count apps even though they’re technically a tweak) so it’s kind of a bummer.

**Comment by clubby789** [comment]
I would have thought a high volume of 'not working' reports would already be enough to know people want it working again.

**Comment by fuzepod** [comment]
> I would have thought a high volume of 'not working' reports would already be enough to know people want it working again.

That's a good point, maybe just a leaderboard of top tweaks and top non-updated tweaks based on the working and broken count would work.

**Comment by jlippold** [comment]
Thats much easier to implement

**Comment by fuzepod** [comment]
A feature to let developers note that they are working on updating the package would also be conveinient. Could have a list of devs who have confirmed they are working on the tweak and just display their GitHub name on the tweakCompatible popup linking to their GitHub or if a second developer confirms have something like a "x Developers" where x is the number of developers working on the tweak and this expands the menu to a list of the developers.  
This would incentivize users to make reports knowing that it will help get their favorite tweaks updated and allow multiple developers to potentially work together more easily.

**Comment by jlippold** [comment]
I do like that idea @fuzepod but I'm not sure how many devs would want to be associated with this. See this back and forth on twitter https://twitter.com/treason/status/978102723634704384

**Comment by JerGisi** [comment]
Good idea 😉😉

**Comment by fuzepod** [comment]
Perhaps the best way to implement this would be to allow developers to link their Cydia packages to their GitHub repo in tweakCompatible so that when users report compatibility their issue is submitted directly to the developer and their simple report is recorded by tweakCompatible as "broken" including their GitHub issue as the "comment" by the bot. It might be important to have the user check if there is already a GitHub issue where they could just comment their bug report and that comment would fill the tweakCompatible bots "comment" to prevent redundant bug reports on the devs repo. All of which is only once a dev has manually confirmed in tweakCompatible that they made the tweak and want these features. An opt out option would be vital..

I think if all these features are optional there is no practical issue here. I understand Andrew Wiik‏ and LaughingQuoll‏s concerns but I think what tweakCompatible does now is not really a bug reporting tool but more of an alterative for a spreadsheet just to check if a tweak supports a new iOS version. However I also think its important to make clear to users that reporting tweak issues in tweakCompatible are not going directly to the developer.

**Comment by fuzepod** [comment]
Wrong thread @GodlyEphesus123 
https://github.com/JunesiPhone/XenInfo/issues/new

**Comment by Vikky7** [comment]
Please update this tweak 👍🏻👍🏻

**Comment by pqhcker** [comment]
Hello. I can't post a review from Cydia.
I don't know if I do something wrong.
Look the video [Video](https://a.uguu.se/BxhrAHFKvLpg_Videodetweak)

**Comment by veteranatgames2** [comment]
Yea this is a good idea yo, I’ve got a few tweaks in mind I would vote for to be updated.
